cli/network.go

```diff
@@ -110,6 +113,9 @@ func initSandboxNetwork(vm *hypervisor.Vm, enc *gob.Encoder, dec *gob.Decoder, p
 			Lo:     false,
 			Bridge: fakeBridge,
 			Ip:     info.Ip,
+			Name:   info.Name,
+			Mac:    info.Mac,
```

Comment:
So nic in vm and nic in netns will share the same mac address, I think this is ok, can you check if this will cause some problem in network test?

Reply:
I'll keep tracking of this, so far we didn't see any harm